cmd/crank/render/render_test.go

```diff
@@ -877,7 +877,7 @@ func TestRender(t *testing.T) {
 					},
 					Requirements: map[string]fnv1.Requirements{
 						"test": {
-							Resources: map[string]*fnv1.ResourceSelector{
+							ExtraResources: map[string]*fnv1.ResourceSelector{
```

Comment:
I think this is broken in main after we merged https://github.com/crossplane/crossplane/pull/6682 earlier today.